text/pl/chapters/coding-compression.md

```diff
@@ -1,51 +1,51 @@
-# Coding - Compression
+# Kodowanie - Kompresja
 
-## What's the big picture?
+## ??What's the big picture?
 
-Data compression reduces the amount of space needed to store files. If you can halve the size of a file, you can store twice as many files for the same cost, or you can download the files twice as fast (and at half the cost if you're paying for the download).
```

Comment:
Jak coś zobaczysz nie tak to możesz też dodawać komentarze do poszczególnych linii.